android/src/main/java/com/uport/signer/RNUportSignerPackage.kt

```diff
@@ -32,4 +32,7 @@ class RNUportSignerPackage : ReactPackage {
         return emptyList()
     }
 
+    override fun createJSModules(): MutableList<Class<out JavaScriptModule>> {
```

Comment:
This method is not needed on recent versions of RN and breaks builds

Reply:
I'm confused about this,  the class `RNUportSignerPackage` implements the interface `ReactPackage` and is necessary to implement all their methods. This don't happends with abstract clases but isn't no case:

  ```java
public interface ReactPackage {
  List<NativeModule> createNativeModules(ReactApplicationContext reactContext);
  List<Class<? extends JavaScriptModule>> createJSModules();
  List<ViewManager> createViewManagers(ReactApplicationContext reactContext);
}```

Reply:
Right, but that interface has changed in recent versions of react native so it would break apps that use them.
You're seeing a warning/error about that method because Android Studio is resolving an older version of react native as dependency, most likely out of mavenCentral instead of `node_modules`.
They now expect to be distributed through `node_modules` installation.
Try running `yarn install` in the root folder of this project to populate node_modules and get an updated version that no longer shows that method in the interface.

Reply:
Thank you so much @mirceanis for the explanation, I confirm that is exactly what you described